packages/types/src/payment-types.ts

```diff
@@ -83,18 +83,24 @@ export enum BALANCE_ERROR_CODE {
   VERSION_NOT_SUPPORTED,
 }
 
+export interface IPaymentNetworkBaseEvent {
+  name: EVENTS_NAMES;
```

Comment:
```suggestion
  name: EVENTS_NAMES;
  contractEvents?: ESCROW_EVENTS_NAMES;
```

Reply:
is this necessary? same as before, it's a bit strange. Ideally EVENT_NAMES would be passed as a generic:

```
export interface IPaymentNetworkBaseEvent<TEventNames =  EVENT_NAMES> {
  name: TEventNames
}
```